modules/@apostrophecms/notification/index.js

```diff
@@ -259,8 +259,17 @@ module.exports = {
               throw self.apos.error('invalid');
             }
             start = Date.now();
-            modifiedOnOrSince = req.query.modifiedOnOrSince && new Date(req.query.modifiedOnOrSince);
-            seenIds = req.query.seenIds && self.apos.launder.ids(req.query.seenIds) && req.query._id.split(',_id=');
+
+            try {
+              modifiedOnOrSince = req.query.modifiedOnOrSince && new Date(req.query.modifiedOnOrSince);
+            } catch (e) {
+              throw self.apos.error('invalid');
+            }
+            seenIds = req.query.seenIds && self.apos.launder.ids(req.query.seenIds);
+            if (seenIds) {
+              seenIds = req.query.seenIds.split(',_id=');
```

Comment:
OK I see, you're formatting it like:

"seenIds=xxx,id=yyyy,id=zzz"

Entity escaping probably makes this work without ambiguity, but the convention would be just:

seenIds=xxx,yyy,zzz"

And I think that would be less confusing.

Or, you could take advantage of the PR of mine that just got merged yesterday, and just pass the array value for seenIds directly to `apos.http.get`. That is supported now and it is sent in a format that will be unpacked as an array automatically for the seenIds parameter server side:

```
// This is allowed now without manual joining and splitting
await apos.http.get('http://whatever', { qs: { seenIds: [ 'an', 'array' ] } }, modifiedSince: 'something' });
```

Reply:
Cool, I took advantage of it. Thanks for the tip.